src/chapters/jdbc-auth.xml

```diff
@@ -201,11 +201,11 @@ upgrade-pre-0.9.6.sql upgrade-pre-0.9.8.sql</computeroutput>
 <prompt>Enter password:</prompt> <userinput><replaceable>password</replaceable></userinput>
 <prompt>$</prompt></screen>
                 </informalexample>
-                <para>If you are upgrading from a version of Guacamole before 0.9.9, schema changes
+                <para>If you are upgrading from a version of Guacamole before 0.9.10-incubating, schema changes
```

Comment:
Sorry, but we need to add a *new* section for the 0.9.10-incubating upgrade script. Changing the 0.9.9 section to instead point to 0.9.10-incubating makes it wrong. :(